acvm/src/pwg/brillig.rs

```diff
@@ -18,6 +21,7 @@ impl BrilligSolver {
         initial_witness: &mut WitnessMap,
         brillig: &Brillig,
         bb_solver: &B,
+        acir_index: usize,
```

Comment:
Spoke on huddle, can create an issue or followup PR for this to make it the same with all solvers